src/tv2_afvd_studio/config-manifests.ts

```diff
@@ -315,8 +315,8 @@ export const studioConfigManifest: ConfigManifestEntry[] = [
 	},
 	{
 		id: 'SwitcherSource.SplitArtKey',
-		name: 'Video Switcher Split Screen Art Key',
-		description: 'Video Switcher input for Split Screen Art Key',
+		name: 'ATEM Split Screen Art Key',
+		description: 'ATEM input for Split Screen Art Key (for TriCaster see documentation)',
```

Comment:
If this is ATEM specific, should the id then still be `SwictherSource...`?
Same goes for the other configs